requirements_dev.txt

```diff
@@ -1,10 +1,759 @@
--r requirements.txt
+#
```

Comment:
there isn't much diff/ between dev and requirements file if it produces a compiled version which includes packages from requirements.

The dev tools keep getting updated and maybe not required for all instances while working on codebase. generally useful only when someone is developing and testing a new feature, they'd dev tools packages.

Reply:
Okay, I will amend the readme file.